### DISCUSSION TITLE
Codespace SSH fails to connect

### DISCUSSION THREAD

**Discussion by bmitchinson** [community/community#30948] [Codespaces]
Hey there. Loving the experience of setting up codespaces and connecting from my mac machines.

Looking at ssh-ing in from Ubuntu on Windows Subsystem Linux, but receiving an error of `codespace@localhost: Permission denied (publickey,password)`

I'm receiving an ssh error that I'm having trouble debugging. I have the SSH key added to my gh account, as prompted by the `gh` cli.

![image](https://user-images.githubusercontent.com/33632286/186961703-98c70da9-8fe2-4ed7-ad3c-30ab08f1eb9b.png)

```bash
❯ ssh-add
Identity added: /home/bmitchinson/.ssh/id_ed25519 (/home/bmitchinson/.ssh/id_ed25519)
❯ ssh-add -l -E sha256
256 SHA256:/WNElS+MUutFjVafi3Ig4c5{.........removed.........} /home/bmitchinson/.ssh/id_ed25519 (ED25519)
❯ gh codespace ssh
? Choose codespace: bmitchinson/notion-publish-tool (main*): furry sniffle
codespace@localhost: Permission denied (publickey,password).
shell closed: exit status 255
```

Any ideas? Thanks so much. Hoping this discussion helps someone else since this error is rarely reported

**Comment by spy653** [comment]
I'm having a similar issue, failing on password despite never having set a password. Did you figure anything out?

**Reply by Freed-Wu** [reply]
Met same error in archlinux.
